### PR TITLE
Ruby version upgrade, etc

### PR DESCRIPTION
**Pull request by knutsenm**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -299,8 +300,9 @@ GEM
     nenv (0.3.0)
     netrc (0.11.0)
     newrelic_rpm (4.8.0.341)
-    nokogiri (1.10.10)
-      mini_portile2 (~> 2.4.0)
+    nokogiri (1.13.6)
```
Comment (knutsenm) [inline Gemfile.lock:303]: I couldn't build the earlier version's native extensions on my M1 MacBook

File: Gemfile.lock [hunk 2]
```diff
@@ -519,6 +522,7 @@ DEPENDENCIES
   awesome_print
   axlsx (~> 3.0.0.pre)
   better_errors
+  bigdecimal (< 2)
```
Comment (knutsenm) [inline Gemfile.lock:525]: https://github.com/ruby/bigdecimal#which-version-should-you-select

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 sudo: false
 language: ruby
-rvm: 2.6.10
+rvm: 2.7.6
```
Comment (knutsenm) [inline .travis.yml:3]: However, Travis has not actually been running

File: Gemfile.lock [hunk 4]
```diff
@@ -591,4 +595,4 @@ DEPENDENCIES
   yard (~> 0.9.5)
 
 BUNDLED WITH
-   2.0.0
+   1.17.3
```
Comment (knutsenm) [inline Gemfile.lock:598]: Bundler 2 would not behave for me.

File: config/initializers/sidekiq.rb [hunk 5]
```diff
@@ -25,11 +25,8 @@
   config.redis = redis_settings
 end
 
-Sidekiq.default_worker_options = {
```
Comment (knutsenm) [inline config/initializers/sidekiq.rb:28]: `default_worker_options` is deprecated in favor of `default_job_options`

Comment (knutsenm) [reply config/initializers/sidekiq.rb:28]: Oops, not until sidekiq v7 ...

**Review by jesusfreak3 — DISMISSED** [review]

File: config/initializers/factory_girl.rb [hunk 6]
```diff
@@ -1,4 +1,4 @@
-if defined? FactoryGirl
+if defined? FactoryBot
```
Comment (knutsenm) [inline config/initializers/factory_girl.rb:1]: Should rename the file as well

File: Gemfile [hunk 7]
```diff
@@ -65,37 +72,44 @@ gem 'docusign_rest', '~> 0.4.4'
...
+  gem 'rails-controller-testing'
+
   gem 'mocha', '~> 1.7.0'                 # Test mocking and stubbing
-  gem 'pry-byebug', '~> 3.6.0'            # Debugger
+  gem 'pry-byebug', '~> 3.9.0'            # Debugger
   gem 'pry-rails', '~> 0.3.5'
-  gem 'rack_session_access', '~> 0.1'     # Edit user-agent session
+  gem 'rack_session_access', '~> 0.2.0'     # Edit user-agent session
   gem 'reek', '~> 5.2'                    # Linter
   gem 'rubocop', '~> 0.60'                # Linter
```
Comment (knutsenm) [inline Gemfile:107]: We like to use `standard` now.

Comment (jesusfreak3) [reply Gemfile:107]: so another tasks is moving away from CAS and to OKTA, so this will be handled during that process.  Does that make sense?

**Comment by knutsenm** [comment]
Closing in favor of #252

**Pull request closed without merging**